omrmakefiles/rules.mk

```diff
@@ -206,12 +206,15 @@ define CLEAN_COMMAND
 endef
 endif
 
+DDR_SED_COMMAND := \
+    sed -n -e '/^DDRFILE_BEGIN /,/^DDRFILE_END /s/^/@/p'
```

Comment:
I thought the point of this was to stop producing files with `@` characters?

Reply:
Ignore my last comment: we still want the `@` characters in `*.i` files otherwise we'd have to change `MacroTool.cpp`.

Reply:
We just can't have the `getmacros` script write `@` characters into the `.cpp` source files, since that's what xlC consumes. The rest of the pipeline still expects the macro info scraped from the annotated and preprocessed source to have `@` characters to begin lines.